test/extension/load_extension.test

```diff
@@ -53,11 +54,12 @@ QUACK
 statement error
 QUAC
 ----
-<REGEX>:.*Parser Error:.*
+<REGEX>:Parser Error: Did you.*
```

Comment:
I think this can be "Did you mean.*". If you look in the code you can see that those are the hardcoded words.

Reply:
Yes, thank you. That way would be better